notifications-service/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+
+
+version: '3.8'
+
+services:
+  zookeeper:
```

Comment:
@SKuzmanoski and team, let's move the docker-compose file to project's root (root dir), then reference the `notifications-service` build with `build: notifications-service` instead of `build: .`, because all teams should use the same docker-compose, otherwise the scaffold looks ok ✅

Reply:
also I'm getting the following error:
```
 => CACHED [2/3] WORKDIR /app                                                                                                                                    0.0s
 => ERROR [3/3] COPY target/notifications-service-0.0.1.jar .                                                                                                    0.0s
------
 > [3/3] COPY target/notifications-service-0.0.1.jar .:
------
failed to solve: failed to compute cache key: "/target/notifications-service-0.0.1.jar" not found: not found
```

try implementing a multi stage Dockerfile
https://docs.docker.com/build/building/multi-stage/
where the first stage will produce the .jar and the second stage will copy and run the jar as you have it currently.

Reply:
It should work now, thank you for the provided reference.

Reply:
Works now, thanks!